templates/web/common/includes/producers_platform_top_panel.tt.html

```diff
@@ -0,0 +1,56 @@
+<!-- start templates/[% component.name %] -->
+
```

Comment:
I think we should add more comments in templates using the `[%# comment %]`

```suggestion
[%# this template is included on home page, it helps redirect producers to pro platform %] 
```

Reply:
Good idea!

Reply:
`[%# this template is included on home page, it helps redirect producers to pro platform %] 
`

In fact it does not work, it's currently not shown on the public platform.